### DISCUSSION TITLE
Maintaining the brew-cask image

### DISCUSSION THREAD

**Discussion by mario-fehr** [Sequel-Ace/Sequel-Ace#630] [General] [answered]
Hi guys it's hard to keep the brew-cask up to date when the package path or package name is always changing. If I always have to adjust it, the release takes longer due to manual review. @Kaspik @Jason-Morcos

**Comment by Kaspik** [comment, marked as answer]
ugh, sorry, I missed that previously it had `-release` suffix. We will stick with what we currently do:
- `Sequel-Ace-x.x.x.zip` for releases - there is no reason to append anything
- `Sequel-Ace-x.x.x-betax.zip` for betas
- `Sequel-Ace-x.x.x-rcx.zip` for release candidates

The tagging is I believe also the same for last months:
- `beta/x.x.x-buildNumber` for beta and release candidates
- `production/x.x.x-buildNumber` - for production

Thanks for pointing this out.

**Comment by Kaspik** [comment]
Update the wiki as well: https://github.com/Sequel-Ace/Sequel-Ace/wiki/Release-manager,-Changelog-and-Submission

**Comment by mario-fehr** [comment]
@Kaspik thanks for your answer I will add the new structure in the cask,

**Comment by mario-fehr** [comment]
@Kaspik one last thing 3.0.0 3006 is marked as prerelease. I think this should be release.

**Comment by Kaspik** [comment]
We are waiting for review on AppStore, until we release it's still pre-release phase of the final release. :) It will become real release in the moment once we are approved and ship it! 🚢
